src/.goreleaser.yml

```diff
@@ -24,11 +24,16 @@ brews:
     license: "MIT"
     folder: Formula
     dependencies:
+      - name: go
       - name: jq
```

Comment:
I think we can specify versions here?  should we?

Reply:
Yes, I think we need to. I'll look into it.